components/rating.css

```diff
@@ -0,0 +1,14 @@
+@import url(/stylesheets/main.css);
+
+.ratings-container{
+    width: 30rem;
+    background-color: hsl(var(--primary-lighter));
```

Comment:
No need of defining hsl this way if you're not dealing with opacity.
Can simple say background-color: var(--primary-lighter);

Reply:
if I declare the variables directly in hsl then I wont be able to use them while i need different opacity.